src/components/Checklist.js

```diff
@@ -0,0 +1,21 @@
+import '../styles/global.css'
+
+function Checklist() {
+    // TODO: use the useState hook to replace the checklist array
+    const checkList = ["hat", "sunglasses", "water"];
+    return (
+        <div className="checklist-card">
+            <h1 className="list-header">What to Pack</h1>
+            <ul >
+                {checkList.map((item, index) => (
+                    <div key={index}>
+                        <input value={item} type="checkbox" />
+                        <span className="list-item">{item}</span>
+                    </div>
```

Comment:
Let's have an exercise to add items to the checklist! Do you think it would be a good idea? That could be a good exercise so they can great a button from scratch and practicing using the state setter from useState